tests/app/api/services/test_suppliers.py

```diff
@@ -182,6 +182,51 @@ def test_get_expired_licences_returns_vic_only(self, supplier, users):
             }
         ]
 
+    @pytest.mark.parametrize(
+        'supplier', [
+            {
+                'labourHire': {
+                    'sa': {
+                        'expiry': pendulum.today(tz='Australia/Sydney').add(days=10).format('%Y-%m-%d'),
+                        'licenceNumber': 'S123456'
+                    },
+                    'vic': {
+                        'expiry': pendulum.today(tz='Australia/Sydney').add(days=10).format('%Y-%m-%d'),
+                        'licenceNumber': 'V123456'
+                    }
+                }
+            }
+        ], indirect=True
+    )
+    def test_ignore_sa_expired_licences_and_return_vic_only(self, supplier, users):
+        expiry = pendulum.today(tz='Australia/Sydney').add(days=10).format('%Y-%m-%d')
+        suppliers_with_expired_licences = suppliers.get_suppliers_with_expiring_labour_hire_licences(days=10)
+
+        assert suppliers_with_expired_licences[0]['labour_hire_licences'] == [
+            {
+                'expiry': expiry,
+                'state': 'vic',
+                'licenceNumber': 'V123456'
+            }
+        ]
+
+    @pytest.mark.parametrize(
+        'supplier', [
+            {
+                'labourHire': {
+                    'sa': {
+                        'expiry': pendulum.today(tz='Australia/Sydney').add(days=10).format('%Y-%m-%d'),
+                        'licenceNumber': 'S123456'
+                    }
+                }
+            }
+        ], indirect=True
+    )
+    def test_ignore_sa_expired_licences(self, supplier, users):
```

Comment:
This test and the one above look like they're testing that SA is ignored.  Maybe we only need one?  What do you think?

Reply:
removed test_ignore_sa_expired_licences test